libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1452,7 +1452,7 @@ void WaveTrack::ClearAndPasteOne(
    track.HandleClear(t0, t1, false, split, shiftClipAtT1ToT0);
 
    // And paste in the new data
-   PasteOne(track, t0, src, startTime, endTime, split);
+   PasteOne(track, t0, src, startTime, endTime, !split);
```

Comment:
Reviewing this commit, I looked at all uses of `PasteOne`, and there is one call that was not rewritten, because it passes a defaulted value.

That is in sync-lock adjustment of a WaveTrack to lengthen it with silence.

Was this case tested?  Is the desired behavior to insert silence without splits, or with them?  Is this consistent with other insertions of silence, like the Silence Audio command?

Reply:
The commit that made this change fails to compile because one more use of `split` remains in `WaveTrack::PasteOne`

Reply:
> Reviewing this commit, I looked at all uses of PasteOne, and there is one call that was not rewritten, because it passes a defaulted value.

The default value was negated, and so was its usage in `PasteOne`, so all good here. The intention was from the very start not to change the behaviour for existing usages, so I haven't reconsidered them.